gensim/models/atmodel.py

```diff
@@ -107,96 +134,69 @@ def construct_author2doc(doc2author):
 
 
 class AuthorTopicModel(LdaModel):
-    """
-    The constructor estimates the author-topic model parameters based
```

Comment:
Need end2end example of usage in docstring (i.e. I can  copy-paste it and this should works

Reply:
Done at the end of the docstring as a separate section.